.github/workflows/nodejs.yml

```
@@ -44,4 +51,4 @@ jobs: # each workflow consists of 1+ jobs; by default, all jobs run in parallel
         run: yarn typechain
 
       - name: Run hardhat tests
-        run: yarn test:hardhat
+        run: yarn test:hardhat
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)